packages/foreman/rubygem-unicode-display_width/rubygem-unicode-display_width.spec

```diff
@@ -2,7 +2,7 @@
 %global gem_name unicode-display_width
 
 Name: rubygem-%{gem_name}
-Version: 1.8.0
+Version: 1.6.1
```

Comment:
Going back form 1.8.0 to 1.6.1 is weird. Needs some investigation.

Reply:
Now I remember: hammer has an unbounded dependency on it, but Rubocop also ends up pulling it in which then creates a constraint on an older version.

Reply:
fixed via https://github.com/theforeman/smart-proxy/pull/851